src/Cookie.php

```diff
@@ -82,11 +82,8 @@ public function fetch(string $name)
             return null;
         }
         try {
-            /** @var string $stored */
+            /** @var string $stored */ 
             $stored = $_COOKIE[$name];
```

Comment:
revert. `$_COOKIE[$name]` can be an array. in that case, casting to a string will result in a notice + unexpected result.

https://3v4l.org/u4X5h